tests/common.py

```diff
@@ -0,0 +1,124 @@
+    realpath_directory = os.path.realpath(
+        os.path.join(os.getcwd(), os.path.dirname(__file__)))
+    plan_dir = os.path.join(realpath_directory, 'integration/queries/tpch_substrait_plans')
+    plan_path = os.path.join(plan_dir, filename)
+
+    f = open(plan_path, 'r')
+    plan = json.loads(f.read())
+
+    return json.dumps(plan)
+
+
+class SubstraitUtils(object):
+    """
+    Common utility for substrait integration tests.
+    """
+    def __init__(self, name="SubstraitUtils"):
+        """
+
+        """
+        self.logger = LOGGER
+        self.logger.info(name)
+
+    @staticmethod
+    def arrow_sort_tb_values(table: type[Table],
+                             sortby: list[str]) -> type[Table]:
+        """
+        Sort the pyarrow table by the given list of columns.
+
+        Args:
+            table: Original pyarrow Table
+            sortby: Columns to sort the results by
+
+        Returns:
+            Pyarrow Table sorted by given columns
+
+        """
+        table_sorted_indexes = pa.compute.bottom_k_unstable(table,
+                                                            sort_keys=sortby,
+                                                            k=len(table))
+        table_sorted = table.take(table_sorted_indexes)
+        return table_sorted
+
+    @staticmethod
+    def format_sql_query(sql_query: str, file_names: list[str]) -> str:
+        """
+        Replace the 'Table' Parameters from the SQL query with the relative
+        file paths of the parquet data.
+
+        Args:
+            sql_query: SQL Query
+            file_names: List of file names
+
+        Returns:
+            SQL Query with file paths
+        """
+        sql_commands_list = (
+            [line.strip() for line in sql_query.strip().split("\n")]
+        )
+        sql_query = " ".join(sql_commands_list)
+        # Get full path for all datasets used in the query
+        parquet_file_paths = get_full_path(file_names)
+
+        return sql_query.format(*parquet_file_paths)
+
+    @staticmethod
+    def format_substrait_query(substrait_query: str,
+                               file_names: list[str]) -> bytes:
+        """
+        Replace the 'local_files' path in the substrait query plan with
+        the full path of the parquet data.
+
+        Args:
+            substrait_query: Substrait Query
+            file_names: List of file names
+
+        Returns:
+            Substrait query plan in byte format
+        """
+        # Get full path for all datasets used in the query
+        parquet_file_paths = get_full_path(file_names)
+
+        # Replace the filename placeholder in the substrait query plan with
+        # the proper parquet data file paths.
+        for count, file_path in enumerate(parquet_file_paths):
+            substrait_query = substrait_query.replace(
+                f"FILENAME_PLACEHOLDER_{count}", file_path)
+
+        return tobytes(substrait_query)
```

Comment:
```suggestion
        return pa.lib.tobytes(substrait_query)
```
(since I removed the import above)